pom.xml

```diff
@@ -169,6 +169,13 @@
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-support</artifactId>
     </dependency>
+    <dependency>
+      <groupId>de.dagere.peass</groupId>
+      <artifactId>dependency</artifactId>
+      <version>0.2.3-SNAPSHOT</version>
```

Comment:
Please always use the latest and same Peass as the rest of the project (in this case: `${peass.version}`).

Reply:
I changed peass-version for dependency-module to ${peass.version}.